{"name": "test_publish_article", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message:", "trace": "self = <scripts.mp.test_publish_article.TestPublishArticle object at 0x000001CC86699470>\n\n    @allure.severity(allure.severity_level.CRITICAL)\n    def test_publish_article(self):\n>       self.home_proxy.go_publish_page()   # 跳转到发布文章页面\n\nscripts\\mp\\test_publish_article.py:42: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\npage\\mp\\home_page.py:60: in go_publish_page\n    self.home_handle.click_content_manage()\npage\\mp\\home_page.py:42: in click_content_manage\n    self.home_page.find_content_manage().click()\npage\\mp\\home_page.py:24: in find_content_manage\n    return self.get_element(self.content_manage)\nbase\\mp\\base.py:14: in get_element\n    element = wait.until(lambda x: x.find_element(*location))\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"3d32a00d8d52bf619e00894567b45dfc\")>\nmethod = <function BasePage.get_element.<locals>.<lambda> at 0x000001CC866AA158>, message = ''\n\n    def until(self, method, message=''):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value is not False.\"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.time() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, 'screen', None)\n                stacktrace = getattr(exc, 'stacktrace', None)\n            time.sleep(self._poll)\n            if time.time() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message:\n\nd:\\programs\\python\\python36\\lib\\site-packages\\selenium\\webdriver\\support\\wait.py:80: TimeoutException"}, "attachments": [{"name": "log", "source": "cc5e74c2-fe21-40cb-b234-67143b3a333d-attachment.txt", "type": "text/plain"}], "start": 1594866390607, "stop": 1594866400795, "uuid": "a0ccde64-4a4b-46ef-819d-20d09f6ed522", "historyId": "a9043702b48e54250431ac4e5855384d", "testCaseId": "a46bb9f9509cb1ae5dd0a4bd99b23530", "fullName": "scripts.mp.test_publish_article.TestPublishArticle#test_publish_article", "labels": [{"name": "severity", "value": "critical"}, {"name": "tag", "value": "run(order=1)"}, {"name": "parentSuite", "value": "scripts.mp"}, {"name": "suite", "value": "test_publish_article"}, {"name": "subSuite", "value": "TestPublishArticle"}, {"name": "host", "value": "LAPTOP-KQP6EJ3A"}, {"name": "thread", "value": "10124-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "scripts.mp.test_publish_article"}]}